From 3d44467aa1b5743fe36a7d5fda1067f9d3e6b23d Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Tue, 19 Jan 2021 08:34:23 +0200
Subject: [PATCH] [SW-30207] i2c indirect: clr sts print as dbg

During fw update, preboot recevies multiple CLR STS
commands. Each time printing this msg to console
causes delay in processing next buffers resulting
in firmware update failure. Made this print dbg but
kept other command prints as their frequency is very
less.

Change-Id: Id45999c0f707eef228b633e22f010933afbf71c1
---

diff --git a/preboot/module_fwupd.c b/preboot/module_fwupd.c
index 9dc8314..a3f93c5 100644
--- a/preboot/module_fwupd.c
+++ b/preboot/module_fwupd.c
@@ -162,7 +162,7 @@
 	switch (cmd.cmd) {
 	case HL_UPD_CLR_STS:
 		regmap_writel(HL_UPD_STS_NOOP, mmUPD_STS);
-		preboot_puts("FW UPD CLR STS received\n");
+		preboot_puts_dbg("FW UPD CLR STS received\n");
 		break;
 	case HL_UPD_RST_STATE:
 		module_fwupd_reset_state();
